bahmni-commons-omod/src/main/java/org/bahmni/module/bahmnicommons/web/v1_0/controller/search/BahmniAppointmentsPatientSearchController.java

```diff
@@ -55,8 +55,7 @@ public void setPatientService(PatientService patientService) {
                     searchParameters.getProgramAttributeFieldValue() != null ||
                     searchParameters.getProgramAttributeFieldName() != null ||
                     searchParameters.getAddressSearchResultFields() != null ||
-                    searchParameters.getPatientSearchResultFields() != null ||
-                    searchParameters.getLoginLocationUuid() != null) {
+                    searchParameters.getPatientSearchResultFields() != null) {
```

Comment:
@pedroedrasousa is the login location UUID actually part of the request?

Reply:
@mks-d loginLocationUuid is part of the request as a query string parameter and, when building the request on front, it's being fetched from a session service. It's not part of the query.

![image](https://user-images.githubusercontent.com/68599335/100869378-e400e480-3494-11eb-8518-feec63661fc5.png)